CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 ## 0.8.0-SNAPSHOT (current master)
 
+### breaking changes
+ * removing of `oshdb-util:util.time.TimestampFormatter` ([#419])
```

Comment:
usually we write changelog entries with active verbs, instead of passive (…ing) sentences.

```suggestion
 * remove `oshdb-util:util.time.TimestampFormatter` class ([#419])
```